README.md

```diff
@@ -20,10 +20,7 @@ need a Terraform 0.11.x-compatible version of this module, the last released ver
 ```HCL
 module "gce-lb-http" {
   source            = "GoogleCloudPlatform/lb-http/google"
-  version           = "~> 3.1"
-
   name              = "group-http-lb"
-  project           = "my-project-id"
```

Comment:
Specifying project is a best practice, please keep it.

Reply:
Done.